api/v1alpha1/common.go

```diff
@@ -233,3 +233,32 @@ type Condition struct {
 	// LastTransitionTime describes last time the condition transitioned from one status to another.
 	LastTransitionTime metav1.Time `json:"lastTransitionTime,omitempty"`
 }
+
+type OperationType string
+
+const (
+	DataLoadType    OperationType = "DataLoad"
+	DataBackupType  OperationType = "DataBackup"
+	DataMigrateType OperationType = "DataMigrate"
+	DataProcessType OperationType = "DataProcess"
+)
+
+type OperationRef struct {
```

Comment:
How about using CrossVersionObjectReference to make it possible for more cases in future. https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/autoscaling/v1/types.go#L27